packages/db-schema/src/indexing/defs.ts

```diff
@@ -0,0 +1,204 @@
+import { sql } from "drizzle-orm";
+import {
+  bigint,
+  customType,
+  index,
+  integer,
+  jsonb,
+  pgTable,
+  timestamp,
+  uniqueIndex,
+  uuid,
+  varchar,
+} from "drizzle-orm/pg-core";
+
+export { indexingEvents, stakes, stakeChanges };
+
+const bytea = customType<{
+  data: Uint8Array | Buffer; // what your app uses
+  driverData: Buffer; // what node-postgres returns
+  notNull: false;
+  default: false;
+}>({
+  dataType: () => "bytea",
+  toDriver: (v) => (v instanceof Buffer ? v : Buffer.from(v)),
+  fromDriver: (v) => v, // Buffer
+});
```

Comment:
Saw this used in other definition files as well. Should we create a file for custom drizzle types that can be shared?